### DISCUSSION TITLE
How to show a built-in browser error message with shouldUseNativeValidation: true?

### DISCUSSION THREAD

**Discussion by auwtch** [react-hook-form/react-hook-form#7193] [Q&A] [answered]
I want to show a native error message by browser, like this
![image](https://user-images.githubusercontent.com/4533828/144224829-1e80f3ab-32b2-425e-b127-b9e9fb6b6d2c.png)

This one shows browser error
```tsx
<input
  required
  {...form.register(props.name)}
/>
```
But this one does not (react-hook-form way)
```tsx
<input
  {...form.register(props.name, { required: true })}
/>
```

**Comment by bluebill1049** [comment, marked as answer]
if you are using hook form `required`, then you should supply a message instead. assume that you want to customise the message.

**Reply by auwtch** [reply]
Thank you for such a lightning-quick response! But that's the thing, I don't want to customize the message, but show the standard browser message, as in the screenshot. As I understand it is not possible?

**Reply by bluebill1049** [reply]
```
<input
  required
  {...form.register(props.name)}
/>
```
then that's all you need.

**Reply by auwtch** [reply]
Hm, but then formState.errors will be empty, since form.onSubmit won't be triggered

**Reply by bluebill1049** [reply]
if you are using native form validation why do you still need to refer to `errors`?

**Reply by auwtch** [reply]
Actually you are right, I am just exploring an integration with html5 capabilities. Thanks for your help!
